ui/components/app/name/name-details/formatted-value.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { memo } from 'react';
```

Comment:
this new file wasn't necessary, but its useful because we were previously always computing both a "formattedValue" and a "shortenedName", when we only ever used one or the other.